Hugo Cullinan/week01/js-homework-1.js

```diff
@@ -0,0 +1,99 @@
+//----------PART 1 ----------
+//Write a function called squareNumber that will take one argument (a number), square that number, and return the result.
+//It should also log a string like "The result of squaring the number 3 is 9."
+const squareNumber = function(num){
+  const squaredNum = num*num;
```

Comment:
very minor thing, but for readability reasons this line would look better like: 
`const squareNumber = function (num) {`